zircon.core/src/commonMain/kotlin/org/hexworks/zircon/api/TableColumns.kt

```diff
@@ -0,0 +1,60 @@
+package org.hexworks.zircon.api
+
+import org.hexworks.cobalt.databinding.api.binding.bindTransform
+import org.hexworks.cobalt.databinding.api.property.Property
+import org.hexworks.cobalt.databinding.api.value.ObservableValue
+import org.hexworks.zircon.api.component.Icon
+import org.hexworks.zircon.api.component.Label
+import org.hexworks.zircon.internal.fragment.impl.table.TableColumn
+
+/**
+ * This object provides different presets for [TableColumn]s.
+ */
+object TableColumns {
```

Comment:
Can you please add `@Beta` to this and all other new classes?

Reply:
Done with 500e8cb8